lib/setler/settings.rb

```diff
@@ -28,9 +28,9 @@ def self.method_missing(method, *args, &block)
         super(method, *args, &block)
       else
         method_name = method.to_s
-        if method_name.ends_with?("=")
+        if method_name.end_with?("=")
           self[method_name[0..-2]] = args.first
-        elsif method_name.ends_with?("?")
+        elsif method_name.end_with?("?")
```

Comment:
When running the tests, I ran into the error below before changing this to `end_with?`. `ends_with?` is provided by an ActiveSupport String extension, so I'm sure fine with some requiring/loading in these tests, but I figure it's probably not a bad thing to depend on the stdlib implementation here instead.

```
SettingsTest#test_update:
NoMethodError: undefined method 'ends_with?' for an instance of String
    lib/setler/settings.rb:31:in 'Setler::Settings.method_missing'
    test/settings_test.rb:51:in 'SettingsTest#test_update'
```